h/features/models.py

```diff
@@ -63,6 +80,10 @@ class Feature(Base):
                       default=False,
                       server_default=sa.sql.expression.false())
 
+    featurecohorts = sa.orm.relationship('FeatureCohort',
```

Comment:
That said, I think it would probably make more sense to define this relationship on the `FeatureCohort` class rather than here. Think of it this way: features don't need feature cohorts to work, but the reverse is not true. It's nicer to be able to remove the functionality just by deleting the `FeatureCohort` class than if you have to go and remove relationships from other objects.